apps/game/lib/game/cup.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Cup do
+  @moduledoc """
+  generat a hand of n dices in a cup
+  """
+
+  @spec new(integer) :: [integer]
+  def new(n, acc \\ [])
+  def new(0, acc), do: acc
+  def new(n, acc) do
+    new(n-1, [:rand.uniform(6) | acc])
+  end
```

Comment:
Nice use of recursion! What do you think about doing this with a list comprehension? It doesn't matter either way, but they're pretty nice 1 liners.

For example
```elixir
iex(7)> for x <- [1, 2, 3, 4], do: x * 2
[2, 4, 6, 8]
```